job-service-container/pom.xml

```diff
@@ -151,6 +151,18 @@
             <artifactId>job-service-util</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- Logging -->
+        <dependency>            
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!-- End Logging -->
```

Comment:
I don't think these two should be here, given that this is packaging war files and uber jars.  Did you add them for tests?  If so then they should be test scope.